### PR TITLE
Update react-native-windows--vnext--module-request.md

### PR DESCRIPTION
**Pull request by stmoy**
Add sections that request customer information.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4226)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md [hunk 1]
```diff
@@ -18,5 +18,12 @@ assignees: ''
 - [ ] Issue exists on module repo (if so, [link]())
 <!-- If the community module repo already has an issue tracking adding React Native Support, please check this box and link to it. -->
 
-## Overview of Required Functionality
+## Customer Information
+
+### Which app/customer is requesting this module?
```
Comment (chrisglein) [inline .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md]: These seem redundant. And/or Customer Information needs more of a prompt for what sort of information you'd expect (maybe a Contoso example or whatever).

Comment (stmoy) [reply .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md]: I'm using `## Customer Information` as a header, not to request more information.

Comment (stmoy) [reply .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md]: It's probably clearer with fewer headers. Removed in latest commit.

File: .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ assignees: ''
 - Target version: 
 - [ ] Issue exists on module repo (if so, [link]())
 <!-- If the community module repo already has an issue tracking adding React Native Support, please check this box and link to it. -->
+- [ ] Module has a vCurrent/C# implementation (if so, [link]())
```
Comment (chrisglein) [inline .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md]: vCurrent is essentially an internal codename. I don't think it'd necessarily be understood by people outside of our internals. Do we have a better term for this?

Comment (kmelmon) [reply .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md]: We've started calling vcurrent "legacy"

Comment (stmoy) [reply .github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md]: I'll swap from vCurrent to "Legacy".

**Pull request merged**